ctp2_code/gfx/spritesys/FacedSprite.cpp

```diff
@@ -431,7 +431,7 @@ void FacedSprite::DrawDirect(aui_Surface *surf, sint32 drawX, sint32 drawY, sint
 
 
 
-void FacedSprite::DirectionalDraw(sint32 drawX, sint32 drawY, sint32 facing,
+void FacedSprite::DirectionalDraw(sint32 drawX, sint32 drawY, uint32 facing,
```

Comment:
Should this change not more general be executed? The caller may still use a signed integer. The method shown in this review 'Draw' still uses a sint32 for facing. I would not do this so local as it will confuse new contributors if it sometimes 'sint32' and sometimes 'uint32'. In case it was really needed for a compiler issue, can a comment be added why this has been changed here but not in other places.

Reply:
This change was incorrectly including in this series as it was part of wider changes I was making attempting to reduce uninteresting warnings so that I could find actual issues - this patch has now been removed.